docs/index.rst

```diff
@@ -1,11 +1,75 @@
-Girder: High-Performance Data Management
-========================================
+Girder: A data management platform
+==================================
 
-For a high-level overview of the system, we recommend checking out the :doc:`user-guide`.
+What is Girder?
+---------------
+
+Girder is a free and open source web-based **data management platform**. What does
+that mean? Girder is both a standalone application as well as a platform on top of
+which new web services can be quickly and easily constructed. It's meant to be a
```

Comment:
I'm a bit torn here--I really like the overall clarity and tone of this approach, as it seems fact-based and relatively cruft-free.  This sentence about "quickly and easily constructed" seems the most like marketing text to me.  Perhaps even changing/merging it with the following sentence could help?  The word "meant" in the following sentence shows the intent for Girder, so applying something like that to "quickly and easily" shows our goals for the project and makes this seem less like marketing.

This is a fairly small quibble (4 nits at most).